setup/README.md

```diff
@@ -124,134 +172,83 @@ If you were just trusting everything someone told you, you wouldn't be here look
+  Root Key:   6b35860633a0cf852670fd9b5c12ba068875f3804d6711feb16fcd74c723c816
 ```
 
-You should see the deployment being rejected with an error similar to
+Note that there is only trust data for the `signed` tag, not for `unsigned`. Let's try to deploy our fresh images to the cluster. We start with the signed image:
 
 ```bash
-service/sample created
-Error from server: error when creating "STDIN": admission webhook "connaisseur-svc.connaisseur.svc" denied the request: could not find signed digest for image "docker.io/testingconny/testimage:unsigned" in trust data.
+kubectl run signed --image=${IMAGE_PATH}:signed
 ```
 
-> Note that while the container is blocked Kubernetes still creates the service, because it does not reference an image and is thus not denied by Connaisseur. You can clean it up by executing `kubectl delete service sample`.
+You should be prompted that the deployment was successful:
+
+```bash
+pod/signed created
+```
 
-Finally, make sure that Connaisseur will deploy the signed image and isn't just rejecting all images:
+Finally, let's make sure that Connaisseur will not just allow deployment of any image:
 
 ```bash
-kubectl create -f - << EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: sample-deployment
-  labels:
-    app: sample
-spec:
-  selector:
-    matchLabels:
-      app: sample
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        app: sample
-    spec:
-      containers:
-      - name: sample
-        image: ${IMAGE_PATH}:signed
-        imagePullPolicy: Always
-        ports:
-        - containerPort: 5000
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: sample
-  labels:
-    app: sample
-spec:
-  ports:
-  - name: port
-    port: 8080
-    targetPort: 5000
-  selector:
-    app: sample
-  type: LoadBalancer
-EOF
+kubectl run unsigned --image=${IMAGE_PATH}:unsigned
 ```
 
-This deployment should be accepted with
+You should see the deployment being rejected with an error similar to:
 
 ```bash
-deployment.apps/sample-deployment created
-service/sample created
+Error from server: admission webhook "connaisseur-svc.connaisseur.svc" denied the request: could not find signed digest for image "docker.io/connytest/testimage:unsigned" in trust data.
 ```
 
-Your signed images were allowed through, in contrast to those unsigned ones with unknown content. DCT, not in a nutshell, but in Kubernetes!
+or if you pushed the unsigned image to another registry altogether:
+
+```bash
+Error from server: admission webhook "connaisseur-svc.connaisseur.svc" denied the request: no trust data for image "docker.io/connytest/testimage:unsigned".
+```
 
-### 2. Cleanup
+> Note that while deployment of containers is blocked Kubernetes will still create services or other resources you might specify in a deployment.yaml, because those does not reference an image and are thus not denied by Connaisseur. You might have to clean up denied deployments.
```

Comment:
```suggestion
> Note that while deployment of containers is blocked Kubernetes will still create services or other resources you might specify in a deployment.yaml, because those do not reference an image and are thus not denied by Connaisseur. You might have to clean up denied deployments.
```